contents/blog/hog-story.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Why we built languages no one asked for
+date: 2025-01-14
+author:
+ - danilo-campos
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/working_at_posthog_c7c74d7e65.png
+featuredImageType: full
+tags:
+ - People
+ - Culture
+---
+
+# The language no one asked for
+
+If I had to pick a reason why most people hate most jobs, it’s easy: most jobs want you to turn off your imagination.
+
+Most jobs want you to follow the leader, hit some OKR, do the thing someone wrote into an endless slideshow at the start of a quarter. Most jobs punish you for inconvenient insights and taking initiative.
+
+Most jobs, deep down, want us to be meat robots.
+
+But jobs at PostHog don’t work that way at all. In fact, if you’re deferring to your manager, awaiting direction, you’re failing here. At PostHog, you’re expected to recognize problems as they emerge and solve them directly, without asking anyone’s permission.
+
+This requires an ongoing, healthy relationship with one’s own sense of creativity and vision – which kicks off lots of interesting consequences.
+
+I’ll tell you about my favorite example so far:
+
+A couple years back, Marius Andra – one of our longest-tenured engineers – declared war on JavaScript.
+
+And won.
+
+# PostHog problems
+
+PostHog is a machine that ingests millions of events for each of our customers daily.
+
+These events describe every sort of product behavior, from clicking on buttons to giving up altogether.
+
+This is not a new problem. For as long as the web has existed, its crafters have wondered: who is using my product? How do I get them to do more of what I want? Did my changes help or hurt my goals?
+
+Entire businesses rise and fall to help answer these questions, and even today PostHog competes in a dizzying field of options. To win, PostHog has to do one thing better than everyone:
+
+Make data exploration both intuitive and consequential. We win when we’ve provided the insight for *your* win.
+
+This, it turns out, is a problem difficult enough to kill a company.
```

Comment:
Reading this again, I reckon losing this "PostHog problems" section would improve the flow a lot. If you read the intro and then skip to the "You can't just hand out SQL access", it feels like a really natural progression. 

There's useful context in this bit, but maybe we can work the core bits in elsewhere?